src/typedefs.js

```diff
@@ -212,7 +252,7 @@ const React = require('react');
  *
  * @exports profileComponentProps
  * @typedef {object} profileComponentProps
- * @property {object} [clientProfile] - Contact object with data from profile
+ * @property {object} [clientProfile] - Client object with data from profile
```

Comment:
I'm currently fixing these typedefs in PR #379. For now, think we could leave Line 255 alone and change this back to Contact.

The same could be said about Line 268 with `clientProfile`